src/webgl/p5.Geometry.js

```diff
@@ -87,7 +87,59 @@ p5.Geometry = class Geometry {
 
     this.dirtyFlags = {};
   }
-
+  /**
+   * Removes the internal Colors of p5.Geometry
```

Comment:
Maybe we can add one more sentence after this explaining that this lets you supply new colors with `fill()` before drawing it? And otherwise, `fill()` gets ignored?

Reply:
Yes. I was also thinking of that... Okay going to add this.

Reply:
One other small request: could we add a period at the end of the sentence so it fits with the style of the next bit of text? other than that I think everything looks good!

Reply:
You mean fullstop?